modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-16.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-16_{context}"]
+= Release notes for {pipelines-title} General Availability 1.16
+
+With this update, {pipelines-title} General Availability (GA) 1.16 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-16_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.16:
+
+[id="pipelines-new-features-1-16_{context}"]
+=== Pipelines
+
+* With this update, you can configure the resync period for the pipelines controller. Once every resync period, the controller reconciles all pipeline runs and task runs, regardless of events. The default resync period is 10 hours. If you have a large number of pipeline runs and task runs, a full reconciliation every 10 hours might consume too many resources. In this case, you can configure a longer resync period.
++
+.Example of configuring a resync period of 24 hours
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      deployments:
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-pipelines-controller
+                  args:
+                    - "-resync-period=24h" 
+----
+
+* With this update, when defining a pipeline, you can set the `onError` parameter for a task to `continue`. If you make this setting and the task fails when the pipeline is executed, the pipeline logs the error and continues to the next task. By default, a pipeline fails if a task in it fails. 
```

Comment:
Why is "continue" marked here? Is it an official value? It reads like the value is used as a verb and a marked value. I think dropping the marks would be clearer.

Reply:
`continue ` is the value.